crabs/tracker/utils/tracking.py

```diff
@@ -31,9 +31,9 @@ def extract_bounding_box_info(row: list[str]) -> Dict[str, Any]:
     height = region_shape_attributes["height"]
     track_id = region_attributes["track"]
 
-    frame_number = int(filename.split("_")[-1].split(".")[0]) - 1
+    frame_idx = int(filename.split("_")[-1].split(".")[0]) - 1
```

Comment:
Here we assume the filename of the frame has a frame number (1-based) in it. And we compute the `frame_idx` as `frame_number-1`.

This has a few issues in my opinion:
- we are saving `frame_idx` under the key `frame_number`, which would be confusing.
- if the frames are for a video clip that doesn't start with frame 1, then the computed `frame_idx` from the frame won't make sense here.

For a low effort fix I would just have here:
```python
    frame_number = int(filename.split("_")[-1].split(".")[0])
    return {
        "frame_number": frame_number,
        "x": x,
        "y": y,
        "width": width,
        "height": height,
        "id": track_id,
    }
```
And then ensure that whenever we use the value in this dictionary, we use it as a key `frame_number` (1-based) and not as an index `frame_idx` (0-based).